docs/serving/using-auto-tls.md

```diff
@@ -29,26 +29,26 @@ Knative supports the following Auto TLS modes:
 
 1.  Using HTTP-01 challenge
 
-    - In this type, your cluster does not need to be able to talk to your DNS server. You just 
+    - In this type, your cluster does not need to be able to talk to your DNS server. You just
     need to map your domain to the IP of the cluser ingress.
     - When using HTTP-01 challenge, **a certificate will be provisioned per Knative Service.**
     - **HTTP-01 does not support provisioning a certificate per namespace.**
 
 ## Before you begin
 
-You must meet the following prerequisites to enable auto TLS:
+You must meet the following prerequisites to enable Auto TLS:
```

Comment:
@omerbensaadon I'd like a standard way to refer to things like this - just a note for if you're looking for doc issues

Reply:
Started adding things here: https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0

Reply:
Nice, thanks. There's actually an open issue for this here https://github.com/knative/docs/issues/170 if you want to take that one?
I think we should use guidance from the Kubernetes style guide https://kubernetes.io/docs/contribute/style/style-guide/ and the Google developer's style guide where possible https://developers.google.com/style/word-list?hl=en